pkg/cloudclient/gcp/gcp.go

```diff
@@ -28,8 +28,8 @@ var (
 	log = logf.Log.WithName("gcp_cloudclient")
 )
 
-// Client represents a GCP Client
-type Client struct {
+// GCPClient represents a GCP cloud Client
+type GCPClient struct {
```

Comment:
This change is slightly out of scope. Can this be spun out into a separate refactoring card?

Reply:
Done https://github.com/openshift/cloud-ingress-operator/pull/240